examples/miscellaneous/miscellaneous_ex12/miscellaneous_ex12.C

```diff
@@ -756,28 +777,50 @@ void assemble_shell (EquationSystems & es,
 
         } // end of the quadrature point qp-loop
 
+      if (distributed_load)
+      {
+        //Loop on shell faces
+        for (unsigned int shellface=0; shellface<2; shellface++)
+          {
+            std::vector<boundary_id_type> bids;
+            mesh.get_boundary_info().shellface_boundary_ids(elem,shellface,bids);
```

Comment:
This is exactly what I was hoping for - test coverage!  I'll hold off on merging #956 until I can rebase off these changes and see what happens.